lib/executors.go

```diff
@@ -183,7 +183,7 @@ func (scs *ScenarioConfigs) UnmarshalJSON(data []byte) error {
 	result := make(ScenarioConfigs, len(protoConfigs))
 	for k, v := range protoConfigs {
 		if v.configType == "" {
-			return fmt.Errorf("execution config '%s' doesn't have a type value", k)
+			return fmt.Errorf("scenario '%s' doesn't have a specified executor type", k)
```

Comment:
I would've also renamed configType to executorType, but I guess it's fine either way 

more importantly, I wonder whether we can specify more directly that the missing key is `"executor"`

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1501/commits/5d4946716f588f0c6e78921f1102c49b55683ca1